Update deep review contribution visualization

dhimmel opened the pull request.

dhimmel committed:
Update deep-review submodule on 2018-01-24

Updates to https://github.com/greenelab/deep-review/commit/1db0c13e0d5828d325ac8e1eba523b77bc33a7d1

dhimmel committed:
Rerun 01.contrib-git.ipynb

Update paths to diffs to include content (modern structure) and
sections (legacy structure) directories.

dhimmel committed:
Rerun & update 03.contrib-stats.ipynb

dhimmel committed:
Rerun and update 03.contrib-stats.ipynb

dhimmel commented:
`deep-review-contribution-ridge.png` has gotten a bit tighter in terms of spacing. Think we should increase its height more or not?

agitter commented on analyses/deep-review-contrib/03.contrib-stats.ipynb:
```diff
@@ -201,53 +202,58 @@
    "metadata": {},
    "source": [
-    "Authors who first contributed after May 28, 2017. See [discussion here](https://github.com/greenelab/meta-review/issues/23#issuecomment-338833911)."
+    "Authors who first contributed after January 19, 2017, the date of [journal resubmission](https://github.com/greenelab/deep-review/issues/810#issue-290029515)."
```
Do we still want to emphasize the authors post-initial submission?  I find it useful to show who joined in between v0.9 and v0.10.

I'll finish a full review later.

dhimmel replied:
Done in https://github.com/greenelab/meta-review/pull/28/commits/c70d456cebdc15cc469d07b63476b6cc5b240683

dhimmel commented:
@cgreene I have noticed some odd things with `deep-review-contrib/commits.tsv`. Consider the plots unreliable until these have been diagnosed.

dhimmel commented on analyses/deep-review-contrib/commits.tsv:
```diff
@@ -7,38 +7,59 @@ de37b1863b7bb090c20407584ec6db228aa483b4	Casey Greene	csgreene@upenn.edu	Casey G
 566ce970296c958f288b2520f60398d71fb128e7	Greg Way	gregory.way@gmail.com	GitHub	2016-08-03 14:10:34-04:00	2016-08-03 14:10:34-04:00	Merge pull request #3 from greenelab/date-to-readme	6	1	20c94413b860c72e9c89b68884d3ffba0c636d7a, 3855f39ad1298ca0467cfda9b359859e702ba6e7	0	0	0	0
 48b61db79d588b00c0d70c4299559a8cd913bd3b	Casey Greene	csgreene@upenn.edu	Casey Greene	2016-08-05 10:28:05-04:00	2016-08-05 10:28:05-04:00	add codeclimate markdown configuration	4	0	de37b1863b7bb090c20407584ec6db228aa483b4	0	0	0	0
 26bf5a2829245b1c968c4f80870a172e331314fc	Casey Greene	csgreene@upenn.edu	Casey Greene	2016-08-05 10:34:35-04:00	2016-08-05 10:34:35-04:00	fix codeclimate issues that were identified	5	0	48b61db79d588b00c0d70c4299559a8cd913bd3b	0	0	0	0
+b2059b54aacd4e82b5ecd110649340a442e3b6fc	Casey Greene	csgreene@upenn.edu	Casey Greene	2016-08-05 10:34:35-04:00	2016-08-05 10:34:35-04:00	fix codeclimate issues that were identified	4	0	de37b1863b7bb090c20407584ec6db228aa483b4	0	0	0	0
```
See here how this commit titled "fix codeclimate issues that were identified" seems duplicated:
https://github.com/greenelab/deep-review/commit/26bf5a2829245b1c968c4f80870a172e331314fc
https://github.com/greenelab/deep-review/commit/b2059b54aacd4e82b5ecd110649340a442e3b6fc

dhimmel replied:
Ah I think I know what's happened. These commits were originally part of the Deep Review and were ported to initialize manubot-rootstock as per the commit message in https://github.com/greenelab/manubot-rootstock/commit/59991680e68d3af315203891945f8737f5559d1b:

> History proceeding this commit was generated from greenelab/deep-review@daef821 by running:
> 
> ```sh
> git filter-branch \
>   --index-filter 'git rm --cached --ignore-unmatch -r authors references sections .codeclimate.yml author_contact_approve.md authors.tsv'\
>   --prune-empty -- --all
> ```

Since this command ignored files in the `sections` directory, the new commits (the ones that have appeared as duplicates here) should not have touch the files that are counted for word/character changes.

> @cgreene I have noticed some odd things with deep-review-contrib/commits.tsv. Consider the plots unreliable until these have been diagnosed.

Resolved.

dhimmel committed:
Move author renaming to 01.contrib-git.ipynb

Switch 03.contrib-stats.ipynb to show new authors between
first submission and resubmission.

cgreene reviewed (approved):
This looks good to me 👍

agitter reviewed (approved):
>`deep-review-contribution-ridge.png` has gotten a bit tighter in terms of spacing

The tighter spacing doesn't bother me.
